package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "quiq-chat",
-  "version": "1.7.1",
+  "version": "1.7.2",
```

Comment:
Ah yeah you are right.  I was thinking minor was for changes that may break, and nano for ones that wouldn't. Will fix